spec/std/process_spec.cr

```diff
@@ -69,9 +69,28 @@ describe Process do
     process.wait.exit_code.should eq(1)
   end
 
-  it "raises if command could not be executed" do
-    expect_raises(RuntimeError, "Error executing process:") do
-      Process.new("foobarbaz", ["foo"])
+  it "raises if command doesn't exist" do
+    expect_raises(File::NotFoundError, "Error executing process: 'foobarbaz'") do
+      Process.new("foobarbaz")
+    end
+  end
+
+  pending_win32 "raises if command is not executable" do
+    with_tempfile("crystal-spec-run") do |path|
+      File.touch path
+      expect_raises(File::AccessDeniedError, "Error executing process: '#{path}'") do
+        Process.new(path)
+      end
+    end
+  end
+
+  pending_win32 "raises if command could not be executed" do
```

Comment:
Wait, why is this pending?

Reply:
Because I misinterpreted the error message 😅 
Just sent another commit. Let's see if it works. I thought it was because the exception is different, but actually is because I wasn't escaping the path in the spec. That's actually wrong in `File::Error`. We shouldn't be using `inspect` for this. But I'll fix it in another PR. Also, `expect_raises` shows the expected string escaped, but the received string as is, so is quite misleading.

Reply:
(now one spec still remains pending btw)

Reply:
Yup, but in that case the error is different. It raises without a detailed message and I don't have a Windows right now to test it (I left my external hard drive with the VM at the office before the lockdown 😞)

```
Expected File::AccessDeniedError, got #<IO::Error: Error executing process: 'C:\Users\RUNNER~1\AppData\Local\Temp\cr-spec-b8175688\process\crystal-spec-run': > with backtrace:
```